x-pack/legacy/plugins/uptime/public/components/monitor_details/ml/ml_flyout.tsx

```diff
@@ -64,11 +65,19 @@ export function MLFlyoutView({ isCreatingJob, onClickCreate, onClose, canCreateM
             <em>{labels.TAKE_SOME_TIME_TEXT}</em>
           </p>
         </EuiText>
-
         <EuiSpacer />
       </EuiFlyoutBody>
       <EuiFlyoutFooter>
         <EuiFlexGroup justifyContent="spaceBetween" alignItems="flexEnd">
+          <EuiFlexItem grow={true}>
+            <EuiButtonEmpty
+              style={{ width: 100 }}
```

Comment:
I'm not sure - why do we need to set the width?

Reply:
it was taking half width and hence was pusing cancel button to center, i think maybe i could have achieved this by using justifyContent to flexStart

Reply:
let me check an EUI example to achieve this best using EUI